Source/Core/Core/PowerPC/Jit64/Jit_SystemRegisters.cpp

```diff
@@ -323,6 +323,7 @@ void Jit64::mfspr(UGeckoInstruction inst)
     MOV(64, rax, MDisp(rcx, offsetof(CoreTiming::Globals, fake_TB_start_value)));
     SHR(64, rdx, Imm8(3));
     ADD(64, rax, rdx);
+    OR(64, rax, Imm8(1));
```

Comment:
Thanks, this is just a temporary hack-fix, of course.
At this point we aren't really simulating the MotionPlus and we're making big assumptions about the SDKs RNG. :P

Reply:
You should really leave a source comment there then :P
Does this also apply to Interpreter and ARM?

Reply:
Hack was removed.